docs/user/modelling.md

```diff
@@ -0,0 +1,128 @@
+# Modelling
+
+This page describes how to model various kinds of fields on a `nautobot_ssot.contrib.NautobotModel` subclass.
+
```

Comment:
```suggestion
## Quick Reference

| Type | action | notes |
| ----- | ------ | ----- |
| Normal fields | standard attribute | N/A |
| Custom Fields | f"{field_nam}_custom_field" | need to use custom class, etc. etc. |

```

Reply:
^^ not ready in it's current form, but you should be able to see where I am going with this.

Reply:
Great idea! Put something in there.